### ISSUE TITLE
Full Modal size not working

### ISSUE THREAD

**Issue by dvriv** [rsuite/rsuite#41]
### Bug, feature request, or proposal:

Bug

### What is the expected behavior?

Full Modal

### What is the current behavior?

Small Modal

### What are the steps to reproduce?

Not sure if I am doing something wrong, I copy pasted the code example for Full Modal and it is not working, it is showing a small modal. The other modal sizes work perfectly.

```
class Test extends Component {
  constructor(props) {
    super(props);
    this.state = {
      show: false
    };
    this.close = this.close.bind(this);
    this.open = this.open.bind(this);
  }
  close() {
    this.setState({ show: false });
  }
  open(size) {
    this.setState({ show: true });
  }
  render() {
    return (
      <div className="modal-container">
        <ButtonToolbar>
          <Button size="lg" onClick={this.open}>
            Open
          </Button>
        </ButtonToolbar>
        <Modal full show={this.state.show} onHide={this.close}>
          <Modal.Header>
            <Modal.Title>Modal Title</Modal.Title>
          </Modal.Header>
          <Modal.Body>
            Test
          </Modal.Body>
          <Modal.Footer>
            <Button onClick={this.close} appearance="primary">
              Ok
            </Button>
            <Button onClick={this.close} appearance="subtle">
              Cancel
            </Button>
          </Modal.Footer>
        </Modal>
      </div>
    );
  }
}

export default Test;

```

**Comment by simonguo** [comment]
May I have a look at your rendered DOM?

e.g:

![image](https://user-images.githubusercontent.com/1203827/40399289-1e84c684-5e6f-11e8-86fc-8f7ef1ae0511.png)

**Comment by dvriv** [comment]
![screenshot_2018-05-22_21-00-41](https://user-images.githubusercontent.com/16443376/40399498-41b13d66-5e03-11e8-9bc2-8d330821c82c.png)

**Comment by dvriv** [comment]
Oh I think I found the problem: (Look at width on the css inspector)

![screenshot_2018-05-22_21-05-34](https://user-images.githubusercontent.com/16443376/40399632-05d5efca-5e04-11e8-901f-dec3975afd2a.png)

**Comment by hiyangguo** [comment]
How do you use style files ? CDN or `less` file ？

**Comment by dvriv** [comment]
@hiyangguo I imported the css file on my index.js (I am using create-react-app)
`
import 'rsuite/dist/styles/rsuite.min.css';
`

**Comment by hiyangguo** [comment]
Sorry , it's a bug. You can add this code in your custom style files , and we will fix this bug in next version.
css
```css
.rs-modal-full .rs-modal-dialog {
  width: calc(100% - 120px);
}
```

**Comment by dvriv** [comment]
I see, thanks for the help

**Comment by dvriv** [comment]
Using `size="xl" full` works perfect too

**Comment by hiyangguo** [comment]
Fix in b61f893
